frontend/app/src/utils/user.ts

```diff
@@ -18,3 +19,34 @@ export function buildDisplayName(
 export function trimLeadingAtSymbol(term: string): string {
     return term.length > 0 && term[0] === "@" ? term.substring(1) : term;
 }
+
+export function findSender(
+    senderId: string,
+    users: UserLookup,
+    webhooks: WebhookDetails[],
```

Comment:
Given that the value of this param is _always_ app.selectedChat.webhooks, you could just reference that directly. There is not actually any need (from a reactivity point of view) to pass it in. Not really sure how I feel about that since a large part of me still thinks - yeah but pure functions are just better.

Reply:
I'll leave it as is